### DISCUSSION TITLE
Help with `Kolibri not found`

### DISCUSSION THREAD

**Discussion by Abhishekzod007** [learningequality/kolibri#11975] [Q&A]
![image](https://github.com/learningequality/kolibri/assets/98864225/1c2a44c9-9159-4300-ba59-862d8b5784b1)

I am trying to run devserver 
I have followed instructions provided [here](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#prerequisites)
it says kolibri command not found

**Comment by AlexVelezLl** [comment]
Hi @Abhishekzod007! Did you activate the pyenv environment? With `pyenv activate kolibri-py3.9`

**Reply by Abhishekzod007** [reply]
Hi @AlexVelezLl , Thanks for the reply.
I did activate the pyenv but I still get the same `Kolibri not found`

![image](https://github.com/learningequality/kolibri/assets/98864225/0af4c9da-6bd8-46bd-96a4-2389f708a2bd)

**Reply by AlexVelezLl** [reply]
Did you run the installation commands when you were inside the pyenv environment?
```bash
pip install -r requirements.txt --upgrade
pip install -r requirements/dev.txt --upgrade
pip install -e .
```
Especially the last one.

Could you run them again and verify that there are no errors in the console when you run them?

**Reply by Abhishekzod007** [reply]
Hii @AlexVelezLl I run the commands you told me to run, and it is not showing any error on the terminal but when the page is not loading but it is stuck to it.

**Reply by Abhishekzod007** [reply]
![image](https://github.com/learningequality/kolibri/assets/98864225/a8d4cde3-a228-4bec-a0b1-fea53211bdaa)
this is in console

**Reply by AlexVelezLl** [reply]
It seems that some specific setting in your browser, or some extension is blocking Kolibrí resources. Could you try testing in a browser with factory settings, or in incognito mode without extensions?

**Reply by Abhishekzod007** [reply]
@AlexVelezLl 
I am sorry but I didn't get the point 
I tried to run this on incognito mode but still it is stuck on loading.
Could you please elaborate it more , what to do now.

**Reply by Abhishekzod007** [reply]
Hii @AlexVelezLl  thank you for having a discussion with me. I successfully started the kolibri server and installed the codebase on my local envrionment.
